src/util/queue/api.ts

```diff
@@ -194,9 +198,9 @@ async function editTicket(id: string, ownerID: string, queueID: string, status:
 /**
  * Deletes a ticket with the given ID.
  */
-async function deleteTicket(id: string, queueID: string): Promise<void> {
+async function deleteTicket(id: string, queueID: string, status: TicketStatus, ta: boolean): Promise<void> {
     try {
-        await APIClient.post(`/queues/${queueID}/ticket/delete`, {id});
+        await APIClient.post(`/queues/${queueID}/ticket/delete`, {id, status, ta});
```

Comment:
why is `ta` being passed into this call now?

Reply:
This is a question for @nickyy96

Reply:
I added this field to the API request body because I validate that the request came from a TA on the backend, and I need this field for that check.

Reply:
Removed in 226022b9438ae8ee8a307eb60aa1dfa397535e68 after determining this was unnecessary